odoo/addons/base/tests/test_base.py

```diff
@@ -409,17 +409,17 @@ def test_50_res_partner_commercial_sync(self):
     def test_60_read_group(self):
         title_sir = self.env['res.partner.title'].create({'name': 'Sir...'})
         title_lady = self.env['res.partner.title'].create({'name': 'Lady...'})
-        test_users = [
+        user_vals_list = [
             {'name': 'Alice', 'login': 'alice', 'color': 1, 'function': 'Friend', 'date': '2015-03-28', 'title': title_lady.id},
-            {'name': 'Alice', 'login': 'alice2', 'color': 0, 'function': 'Friend',  'date': '2015-01-28', 'title': title_lady.id},
+            {'name': 'Alice', 'login': 'alice2', 'color': 0, 'function': 'Friend', 'date': '2015-01-28', 'title': title_lady.id},
             {'name': 'Bob', 'login': 'bob', 'color': 2, 'function': 'Friend', 'date': '2015-03-02', 'title': title_sir.id},
             {'name': 'Eve', 'login': 'eve', 'color': 3, 'function': 'Eavesdropper', 'date': '2015-03-20', 'title': title_lady.id},
             {'name': 'Nab', 'login': 'nab', 'color': -3, 'function': '5$ Wrench', 'date': '2014-09-10', 'title': title_sir.id},
             {'name': 'Nab', 'login': 'nab-she', 'color': 6, 'function': '5$ Wrench', 'date': '2014-01-02', 'title': title_lady.id},
         ]
         res_users = self.env['res.users']
-        user_ids = [res_users.create(vals).id for vals in test_users]
-        domain = [('id', 'in', user_ids)]
+        user_ids = res_users.create(user_vals_list)
```

Comment:
Beware of the confusing name, call it `users` as it is a recordset, and not a collection of ids.